spec/system/claims/support/schools/claims/change_claim_on_check_page_spec.rb

```diff
@@ -216,21 +197,25 @@ def when_i_add_training_hours(hours)
   end
 
   def then_i_expect_the_training_hours_to_be_selected(hours)
-    find("#claims-support-claim-mentor-training-form-hours-completed-#{hours}-field").checked?
+    if hours.to_i == 20
```

Comment:
Is it necessary to convert this to an integer here or could we use string comparison?